eva_cttv_pipeline/consequence_type.py

```diff
@@ -3,37 +3,9 @@
 __author__ = 'Javier Lopez: javild@gmail.com'
 
 
-def _process_consequence_type_file_xls(snp_2_gene_file):
```

Comment:
This was removed because the latest file provided by CTTV was in tsv format, so this is now possibly redundant. If they provide a file in xls format in the future this can be added back, but I think the reason this was in xls format initially was because a tsv file was put into xls format. If this is incorrect and they do provide a xls file in the future then the actual data is simple enough to just change to a tsv file, which keeps the code simpler.